tools/llvm-cbe/llvm-cbe.cpp

```diff
@@ -436,7 +437,8 @@ static int compileModule(char **argv, LLVMContext &Context) {
 #else
                                  FileType
 #endif
```

Comment:
More version checks to remove? (which should also fix the location of that comma)

Reply:
Yep - I spotted a few of these that I missed, I already have a change waiting that removes the rest.